packages/frontend/src/index.tsx

```diff
@@ -10,7 +10,17 @@ import { theme } from './styles/createTheme';
 import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
 import './i18n';
 
-const queryClient = new QueryClient();
+const queryClient = new QueryClient({
```

Comment:
Once we start adding more queries, these should be moved to a centralised location instead of in the middle of other code. But that can be done later.